code/modules/research/machinery/_production.dm

```diff
@@ -182,6 +182,8 @@
 			var/datum/bank_account/our_acc = card.registered_account
 			if(our_acc.account_job && SSeconomy.get_dep_account(our_acc.account_job?.paycheck_department) == SSeconomy.get_dep_account(payment_department))
 				total_cost = 0 //We are not charging crew for printing their own supplies and equipment.
+	if(!is_station_level(z) || is_mining_level(z))
+		total_cost = 0
```

Comment:
it should be noted that, literally immediately above this line, there's a block of code that the comments suggest is intended to remove the tax on off-station lathes. it doesn't WORK, I believe (and in fact has the opposite effect of making departmental lathe tax bypasses NOT work off station), but this code should probably be moved there